pkgs/by-name/zo/zoho-mail/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  dpkg,
+  ffmpeg-full,
+  glib,
+  gtk3,
+  wrapGAppsHook3,
+  nss,
+  xdg-utils,
+  nspr,
+  mesa,
+  systemd,
+  libglvnd,
+  libGL,
+  libGLU,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zoho-mail";
+  version = "1.6.5";
+
+  src = fetchurl {
+    url = "https://downloads.zohocdn.com/zmail-desktop/linux/zoho-mail-desktop-lite-installer-x64-v${version}.deb";
+    hash = "sha256-Rt2lPHzxdbf6jjMmCnTg7Fyo/shc7CESisdRYm+HSg4=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    wrapGAppsHook3
+  ];
+
+  dontWrapGApps = true;
+
+  buildInputs = [
+    dpkg
+    ffmpeg-full
+    glib
+    gtk3
+    nss
+    xdg-utils
+    nspr
+    mesa
```

```suggestion
    libgbm
```
Unless the binary needs mesa specifically